### DISCUSSION TITLE
DwC and IPT for GloBI

### DISCUSSION THREAD

**Discussion by seltmann** [Big-Bee-Network/Big-Bee-community-discussion#3] [Q&A] [answered]
@jhpoelen for the Big-Bee GloBI integration page https://www.globalbioticinteractions.org/bigbee/ there are three collections that already had existing repositories. These are: globalbioticinteractions/ucsb-izc, globalbioticinteractions/mcz and globalbioticinteractions/ummzi.

Do these repositories need to be updated with the new [GloBI template information](https://github.com/globalbioticinteractions/template-dataset) or can they be left as is?

**Comment by jhpoelen** [comment, marked as answer]
@seltmann thanks for asking, I'll update them in a minute.

**Reply by jhpoelen** [reply]
Just applied the changes. Please review.

**Reply by seltmann** [reply]
Great, thanks for updating the [Big-Bee page](https://www.globalbioticinteractions.org/bigbee/) on Global Biotic Interactions
